doc/index.rst

```diff
@@ -54,6 +54,51 @@ Your contributions are welcome!  Simply fork the `GitHub repository <https://git
 
 .. _pull request: https://github.com/python-control/python-control/pulls
 
+The following details suggested steps for making your own contributions to the project using GitHub
+
+1. Fork on GitHub: login/create an account and click Fork button at the top right corner of https://github.com/python-control/python-control/.
+
+2. Clone to computer (Replace [you] with your Github username)::
+
+    git clone https://github.com/[you]/python-control.git
```

Comment:
This will not give you write access to your own origin. Use the SSH URL:

```suggestion
    git clone git@github.com:[you]/python-control.git
```

Reply:
only on linux boxes. Windows goes through

Reply:
I stand corrected. They changed it. But I don't think that has anything to do with Windows vs Unix.
https://docs.github.com/en/free-pro-team@latest/github/using-git/which-remote-url-should-i-use

Reply:
ha! i used to have this issue on linux boxes and switched to windows and didn't have this issue probably when they introduces this giving me the wrong impression

Reply:
so, ok as it is? works for me on mac.